lib/utils/theme/app_theme.dart

```diff
@@ -59,7 +59,7 @@ class AppTheme {
             titleTextStyle: txtThemeLight.bodyMedium?.copyWith(
               fontSize: 20,
               color: AppColors.appBarTextColor,
-              fontWeight: FontWeight.w800,
```

Comment:
pls, remove fontWeight and font size from Theme

Reply:
explain more